### PR TITLE
Add setting options for feature requests/bugs #120

### PR DESCRIPTION
**Pull request by dhamanutd**
issue: https://github.com/jorgegil96/All-NBA/issues/120

> I was thinking of adding an "About" section at the bottom of the settings screen (below "Reddit") with the version number and a "Report bug or suggest a feature" item that opens un a dialog with a textbox with a send button

- [x] Create prototype feedback_form_layout.xml for the dialog
- [x] Adding new &lt;PreferenceCategory&gt; and &lt;Preference&gt; in pref_general.xml :
  - New &lt;Preference Category&gt; for About
  - New &lt;Preference&gt; for App Version
  - New &lt;Preference&gt; for Feedback
- [x] edit SettingsFragment.java:
  - Create method appVersion() to get appVersion number.
  - Create method feedbackDialog() to show dialog of feedback form.
  - In onCreate() adding new code for setting app version number to the preference view.
  - In onCreate() adding new code for feedback onClickListener to show the dialog form.
  - Also insert new variable:
```
    public static final String KEY_APP_VERSION = "app_version";
    public static final String KEY_FEEDBACK = "feedback";
```
- [x] edit strings.xml to add:
```
    <string name="about">About</string>
    <string name="key_app_version">app_version</string>
    <string name="key_feedback">feedback</string>
    <string name="report_bug_or_suggest">Report Bugs or Suggest A Feature</string>
    <string name="send">Send</string>
    <string name="hint_feedback_form">Write your suggestion or bug reporting.</string>
```
### Here the screenshot of App where i do the issue:

screenshot_20171221-140817
screenshot_20171221-140821

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -484,6 +484,12 @@
     <string name="quadruple_doubles">Quadruple Doubles</string>
     <string name="five_x_fives">5x5\'s</string>
     <string name="click_to_view_table">Click to view table</string>
+    <string name="about">About</string>
+    <string name="key_app_version">app_version</string>
+    <string name="key_feedback">feedback</string>
+    <string name="report_bug_or_suggest">Report Bugs or Suggest A Feature</string>
```
Comment (jorgegil96) [inline app/src/main/res/values/strings.xml]: Can you change this to "Report bugs or suggest a new feature" (note the non-capitalized letters.

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -484,6 +484,12 @@
...
+    <string name="send">Send</string>
+    <string name="hint_feedback_form">Write your suggestion or bug reporting.</string>
```
Comment (jorgegil96) [inline app/src/main/res/values/strings.xml]: And this to "Tell us what went wrong or what feature you'd like to see next"

File: app/src/main/java/com/gmail/jorgegilcavazos/ballislife/features/settings/SettingsFragment.java [hunk 3]
```diff
@@ -305,4 +329,23 @@ public void onError(Throwable e) {
...
+
+    public void feedbackDialog() {
+        MaterialDialog feedbackDialog = new MaterialDialog.Builder(getActivity())
+                .title(R.string.report_bug_or_suggest)
+                .customView(R.layout.feedback_form_layout, false)
+                .positiveText(R.string.send)
+                .build();
+
+        View view = feedbackDialog.getCustomView();
+        if (view == null) return;
```
Comment (jorgegil96) [inline app/src/main/java/com/gmail/jorgegilcavazos/ballislife/features/settings/SettingsFragment.java]: Are these two lines needed? Remove them if not.

**Comment by dhamanutd** [comment]
@jorgegil96 changes what you requested been pushed. Thanks.

**Comment by jorgegil96** [comment]
Awesome, thank you!

**Pull request merged**